packages/forms/stories/14-FileUpload.stories.tsx

```diff
@@ -107,17 +57,26 @@ const FileWrapper: React.FC<{
     };
   }, []);
 
+  const handleKeyDown = (e: React.KeyboardEvent<any>) => {
+    if (e.keyCode === KEY_CODES.DELETE || e.keyCode === KEY_CODES.BACKSPACE) {
+      e.preventDefault();
+      onRemove();
+    }
+  };
+
   return (
     <FileList.Item key={name}>
-      <StyledFile type={type} isCompact={isCompact} aria-label="File">
-        <StyledSpan>
-          <StyledEllipsis>{name}</StyledEllipsis>
-          <File.Close onClick={onRemove} aria-label="Remove file" />
-        </StyledSpan>
-        <CSSTransition in={uploadProgress < 100} timeout={200} className="progress-bar">
-          <StyledProgress value={uploadProgress} />
-        </CSSTransition>
-      </StyledFile>
+      <File
+        type={type}
+        isCompact={isCompact}
+        onKeyDown={handleKeyDown}
+        aria-label="File"
+        tabIndex={0}
+      >
+        {name}
+        <File.Close onClick={onRemove} aria-label="Remove file" />
```

Comment:
I noticed that modifying the aria-label does not reflect the change in the DOM structure. Is the aria-label prop passed to the correct element?

Reply:
I removed the `aria-label` styled attr that was masking the prop. Furthermore, this demo is not a valid use of `aria-label` – it provides no value to assistive technology. I've removed it and replaced with `title` – which technically is not The Garden Way ™️ , but holds up for a11y scrutiny. We'll provide the appropriate `Tooltip` treatment on TBD website examples.